src/sorting/quicksort.jl

```diff
@@ -0,0 +1,416 @@
+"""
```

Comment:
Let's just call this `src/sorting.jl` for now (unless you were planning to develop other implementation? :slightly_smiling_face:)